### ISSUE TITLE
Value List Item move handle label - Provide a valid label for form fields - (2036570430)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#5621]
### Module: 
 39 Value List and Value List Item 

### Violation: 
 Provide a valid label for the component's handle button

![image](https://user-images.githubusercontent.com/5023024/202570113-ecf06d4d-a057-4e27-a437-3a1307c130f2.png)

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570430 

### Severity: 
 10 

### Description: 
 [Issue]
There are buttons in the "Test 5. Value List with Drag Enabled" and "Test 6. Grouped Value Lists with Drag Enabled" sections without accessible names. However, the button have accessible name when the elements are activated once. Examples include:

- River
- Mountain
- Lake

[User Impact]
Screen reader users will be unable to determine the purpose of these buttons. Speech input users will have difficulty activating them.

[Code Reference] 
 ```
<span class="handle" data-js-handle="" role="button" tabindex="0">
  (...)
</span>
 ``` 

### Note: 
 [Suggestion]
Provide a valid label for form fields. This includes providing accessible names for buttons. The accessible name of a button can be set with internal text, an aria-label attribute, or an aria-labelledby attribute. Good accessible names are both concise and descriptive.

[Compliant Code Example] 
 ```
<span aria-label="River, press space and use arrow keys to reorder content. Current position 1 of 3." class="handle" data-js-handle="" role="button" tabindex="0">
  (...)
</span>
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
We can push this one back for now.

**Comment by geospatialem** [comment]
Closing as the `value-list` components are deprecated and functionality will not be supported at this time. Refer to the AT support effort of https://github.com/Esri/calcite-design-system/issues/7426 for `list` functionality.
